packages/postgraphile-core/__tests__/fixtures/mutations/mutation-delete.graphql

```diff
@@ -62,3 +63,19 @@ fragment deletePersonPayload2 on DeletePersonPayload {
   }
   query { nodeId }
 }
+
+fragment deletePersonPayloadWithEdge on DeletePersonPayload {
+  clientMutationId
+  deletedPersonId
+  personEdge(orderBy: FIRST_NAME_ASC) {
+    node {
```

Comment:
when requesting an edge we should also request the cursor (otherwise what’s the point?)

```suggestion
    cursor
    node {
```